elements/lisk-chain/src/data_access/cache/base.ts

```diff
@@ -48,4 +56,12 @@ export abstract class Base<T> {
 
 		return this.items;
 	}
+
+	public set needsRefill(needRefill: boolean) {
```

Comment:
if we have both setter and getter without mutation, maybe we can just make the property public?

Reply:
As discussed keeping as is just for consistency.